monarchmoney/monarchmoney.py

```diff
@@ -390,6 +390,235 @@ async def get_account_holdings(self, account_id: int) -> Dict[str, Any]:
             variables=variables,
         )
 
+    async def get_budgets(
+        self,
+        start_date: Optional[str] = None,
+        end_date: Optional[str] = None,
+        use_legacy_goals: Optional[bool] = False,
+        use_v2_goals: Optional[bool] = True,
```

Comment:
What happens if you both of these are set to `True` or `False`? Are those valid parameter combinations?

If a condition for validity of the API call is that `use_legacy_goals != use_v2_goals`, I wonder if we can just simplify this to on `use_legacy_goals` parameter that determines the values of both the parameters in the API call.

Reply:
@grablair  This is the result from the query:

(use_legacy_goals=true, use_v2_goals=true): two lists returned.  One list under dictionary key "goals" and another list under dictionary key "goalsV2".  If you have no goals, the list will be empty.
(use_legacy_goals=false, use_v2_goals=false): no lists returned. No dictionary key called "goals" and "goalsV2" respectively.

So all combinations could be valid

Reply:
Great, thank you!